File: 01 - FUNDAMENTOS DE PROGRAMAÇÃO/C/Exercicio 10.c
/*
10 - Leia valores numéricos até que o valor 0 for digitado. Em seguida,
calcule a média dos valores digitados e informe o usuário.
*/


#include<stdio.h>

int main()
{
  float num, soma=0;
  int cont=0;

  printf("\nDigite valores, será encerrado quando for digitado zero: \n");
  do{
    scanf("%f",&num);
    soma += num;
    cont++;
  } while(num!=0);

  printf("\nA media dos numeros digitados excluindo o zero eh: %f",(soma/(float)(cont-1)));
  return 0;
}

File: 01 - FUNDAMENTOS DE PROGRAMAÇÃO/Python/exercicio10.py
"""
10 - Leia valores numéricos até que o valor 0 for digitado. Em seguida, calcule a média dos valores digitados e informe o usuário.
"""

print('Digite valores, será encerrado quando for digitado zero:\n')

soma=0
cont=0
while True:
  num = float(input())
  soma += num
  cont += 1
  if num==0: break;

resultado=soma/(cont-1);
print('\nA média dos numeros digitados excluindo o zero é: '+str(resultado))